content/discrete-math/concepts/proofs/terms/proof-by-induction/proof-by-induction.md

```diff
@@ -0,0 +1,123 @@
+---
+Title: 'Proof by Induction'
+Description: 'A mathematical method to determine the validity of a statement'
+Subjects: 
+  -  'Coputer Science'
+  -  'Mathematics'
+Tags: 
+  -  'Algorithms'
+  -  'Methods'
+  -  'Data Structures' 
+CatalogContent: 
+  -  'Discrete Mathematics'
+  -  'Computer Science'
+---
+
+**`Proof by Induction`** is a method of mathematical proof that establishes a **base case** to develop an **induction hypothesis** that can then be proven by **induction**.
+
+## Base Case
+
+The *Base Case* verifies the problem or statement through a specific initial value or values.(*n* =0, *n* = 1) 
+
+## Induction Hypothesis
+
+The *Induction Hypothesis* assumes that the statement holds for some *k*, or range of values using *k* as a boundary.
+
+- **Weak Induction**: Induction typically based on a specifc assumption.(*P*(n) = *P*(k), or *n* = *k*)
+- **Strong Induction**: Induction typically involving a range or global case of an assumption.(All integers less than or equal to *k*.)
+
+## Inductive Step
+
+The *Inductive Step* proves the statement holds based on the Inductive Hypothesis.
+
+### Example 
+
+Prove by induction that 2^n > 2n
+
+Base case: 
+
+	When n = 3 we have   2^3 > 2 x 3
+			       8 > 6 **TRUE**
+
+Induction Hypothesis:
+
+	Assume the P(k) is correct for some positive integer k.
+
+			     2^k > 2k.
+
+Induction Step: 
+
+	Since P(k) is true for all positive integers greater than 2, P(k+1) is true.
+
+		2^(k + 1) is the same as 2 x 2^k, which is clearly greater than 2 x 2k,which is equivalent 
+		to 2(1 + k) or due to the communative property of mulitplaction is equivalent to 2(k + 1), 
+		for all k > 2. Or, 
+
+			     2^(k+1) = 2 x 2^k > 2 x 2k = 2(k + 1)  for k>2
+		
+		Hence by mathematical induction P(n) is true for all positive integers n > 2.
```

Comment:
````suggestion
```pseudo
  Since P(k) is true for all positive integers greater than 2, P(k+1) is true.

    2^(k + 1) is the same as 2 x 2^k, which is clearly greater than 2 x 2k,which is equivalent 
    to 2(1 + k) or due to the communitive property of multiplication is equivalent to 2(k + 1), 
    for all k > 2. Or, 

        2^(k+1) = 2 x 2^k > 2 x 2k = 2(k + 1)  for k>2
		
    Hence by mathematical induction P(n) is true for all positive integers n > 2.
```
````